application/src/main/java/org/togetherjava/tjbot/commands/basic/SuggestionsUpDownVoter.java

```diff
@@ -0,0 +1,66 @@
+package org.togetherjava.tjbot.commands.basic;
+
+import net.dv8tion.jda.api.entities.Emote;
+import net.dv8tion.jda.api.entities.Guild;
+import net.dv8tion.jda.api.entities.Message;
+import net.dv8tion.jda.api.events.message.guild.GuildMessageReceivedEvent;
+import org.jetbrains.annotations.NotNull;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.togetherjava.tjbot.commands.MessageReceiverAdapter;
+import org.togetherjava.tjbot.config.Config;
+import org.togetherjava.tjbot.config.SuggestionsConfig;
+
+import java.util.Optional;
+import java.util.regex.Pattern;
+
+/**
+ * Listener that receives all sent messages from suggestion channels and reacts with an up- and
+ * down-vote on them to indicate to users that they can vote on the suggestion.
+ */
+public final class SuggestionsUpDownVoter extends MessageReceiverAdapter {
+    private static final Logger logger = LoggerFactory.getLogger(SuggestionsUpDownVoter.class);
+    private static final String FALLBACK_UP_VOTE = "👍";
+    private static final String FALLBACK_DOWN_VOTE = "👎";
```

Comment:
What was the purpose of adding the emote name in the config if we are anyway sticking to 👍  and 👎 here?

Reply:
**FALLBACK**_UP_VOTE

Either take a look at the code, or read the variable name.

It's a fallback, if the emote in the config can't be found, it has a fallback

Reply:
I just thought its kinda fragile to let the bot crash around just because someone deleted an emote in the server when it was not obvious to them that the emote might be used by the bot.

Or if you setup the bot on ur private server and the emotes are not there yet - no need to crash around.

Rather warn + fallback emote.